### PR TITLE
Update startup.sh

### PR DESCRIPTION
**Pull request by wanoo**
Add port sql params BUG #169

### CODE REVIEW

File: startup.sh [hunk 1]
```diff
@@ -9,13 +9,19 @@ function waitForDB() {
   DB_USER=$(awk -F '[/:@]' '{print $4}' <<< $DATABASE_URL)
   DB_PASS=$(awk -F '[/:@]' '{print $5}' <<< $DATABASE_URL)
   DB_HOST=$(awk -F '[/:@]' '{print $6}' <<< $DATABASE_URL)
+  DB_PORT=$(awk -F '[/:@]' '{print $7}' <<< $DATABASE_URL)
```
Comment (kevinpapst) [inline startup.sh:12]: what happens if there is no `$7`?

**Review by Schrolli91 — APPROVED** [review]
LGTM - but not tested

**Review by kevinpapst — COMMENTED** [review]
Line 7 
```
# todo: port is not used atm
```
could/should be removed

**Comment by Schrolli91** [comment]
yes you're right - that todo should be removed now

File: startup.sh (hunk 1 above)
Comment (wanoo) [reply startup.sh:12]: the script test the var DB_PORT and if it's NaN we will use default port

**Comment by fbossiere** [comment]
Can you tell us when this patch will be available on the kimai/kimai2:apache-debian-master-prod docker image ? I saw the latest version was deployed yesterday but not the kimai/kimai2:apache-debian-master-prod

**Pull request merged**